_topic_map.yml

```diff
@@ -222,6 +222,13 @@ Topics:
     File: installing-bare-metal-network-customizations
   - Name: Installing a user-provisioned bare metal cluster on a restricted network
     File: installing-restricted-networks-bare-metal
+- Name: Installing single node OpenShift (SNO)
+  Dir: installing_sno
```

Comment:
Need to add the openshift-webscale distro tag as it is only for that distribution.

Reply:
I will do that after peer review. Nobody can see it when that tag is added.

Reply:
Done.

Reply:
Change title to "Installing on a single node" or similar.

I'm making this suggestion after the feedback in Slack that this is not a new product, but just a specific installation configuration.